2-dev-tools.md

```diff
@@ -0,0 +1,12 @@
+- What's the status code the server sent back?
+  A: 200
+- What HTTP method did the browser use to make the request?
+  A: GET
+- What is the request path?
+  A: /shows/82/episodes
+- What is the first line of the response body?
+  A: [{"id":4952,"url":"http://www.tvmaze.com/episodes/4952/game-of-thrones-1x01-winter-is-coming","name":"Winter is
```

Comment:
almost there, but is that actually the first line?
the whole body of the response comes in one line, so the first line will be the whole body (it may showed to be in more than one line but checking the numbering in dev tools will show it as one line